samples/algorithms/MajoranaQubits/qsharp.json

```diff
@@ -0,0 +1,12 @@
+{
+    "lints": [
+        {
+            "lint": "deprecatedWithOperator",
+            "level": "error"
+        },
+        {
+            "lint": "deprecatedDoubleColonOperator",
+            "level": "error"
+        }
+    ]
```

Comment:
I'm torn on this one... while it's good to show this can be done, since this sample doesn't use any of the new struct syntax that replaces this syntax, these lints are fairly unrelated.

Reply:
Removed. I had copied it from the MultiFileProject sample in the language folder.